_includes/newcomers/newcomers.html

```diff
@@ -0,0 +1,89 @@
+<h3>Welcome Newcomers!</h3>
+
+<p>
+    Whether you are a first-time contributor or an open-source veteran, you are welcome to contribute to and actively
+    engage in the development of projects at Layer5.
+</p>
+<p>
+    Here are some instructions to get you started and if you haven't yet, join the Slack channel to collaborate with the
+    community.
+</p>
+<p>
+    Also, you can check the quick-links below for jumping stright into things.
+</p>
+<table style="margin:0px; padding:0px;width:100%; table-layout: fixed;">
+    <th>Information</th>
+    <th>Guides</th>
+    <th>Issues up for grabs</th>
+    <tr>
+        <td><a href="" target="_blank">Calender</a> | <a href="" target="_blank">Slack</a> | <a href=""
+                target="_blank">GitHub</a> | <a href="" target="_blank">YouTube</a> | <a href=""
+                target="_blank">Drive</a></td>
+        <td><a href="" target="_blank">First-timers' Welcome Guide</a></td>
+        <td><a href="" target="_blank">help-wanted</a></td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td><a href="" target="_blank">Welcome Newcomers!</a></td>
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
+    </tr>
+    <tr>
+        <td><a href="" target="_blank">Weekly Zoom on Thursdays at 8AM Central</a></td>
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
+        <td><a href="" target="_blank">good first issue</a></td>
```

Comment:
https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22good+first+issue%22

Reply:
I forgot to add the links to the table, I will add and commit